frontend/app/src/components/home/Markdown.svelte

```diff
@@ -44,14 +44,22 @@
         });
     }
 
+    function replaceEveryone(text: string): string {
+        return text.replace(/(^|[\s(){}\[\]])@everyone($|[\s(){}\[\]])/g, (_match, _p1) => {
+            return " **@everyone** ";
```

Comment:
Shouldn't "everyone" be translated? Seems a bit odd to have a hard coded English word like that.

Reply:
Nah

Reply:
Why in the future? It's not like it's hard. It seems just obviously wrong to use a hard-coded English word in a translated system.